src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,45 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.6.0
+13 March, 2023
+
+### Deprecation Notice
+- Support for Kubernetes [1.21](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.21.md) has been deprecated and will be removed next release.
+
+### Features
+
+- Add support for Kubernetes [1.24](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md)
+- Update the Okteto CLI version to [2.13.0](https://github.com/okteto/okteto/releases/tag/2.13.0)
+- Default ingress class for ingresses managed by okteto is `okteto-nginx`. All ingresses managed by okteto are migrated automatically during the upgrade
```

Comment:
"All ingresses managed by Okteto will be automatically updated. No action is required from the user"

Also, can we put this as a breaking change?

Reply:
I'll update it. Not sure it's a breaking change as we are migrating things to prevent any issue. In a breaking change I would expect some action to be taken from the user

Reply:
I've removed it from here and added a paragraph at the beginning. Related to this comment: https://github.com/okteto/docs/pull/339/files#r1133865958